app/assets/stylesheets/petitions/_layout.scss

```diff
@@ -74,3 +74,7 @@
 .column-two-thirds {
   @include grid-column(2/3);
 }
+
```

Comment:
2 things:

- If this is only used in admin, let's put it into an admin specific stylesheet like admin/_layout.scss
- The class name describes what it does rather than what it is. Is there another name we can use like `.admin-main` or something?